service/policy/db/attribute_values.go

```diff
@@ -101,6 +101,15 @@ func (c PolicyDBClient) GetAttributeValue(ctx context.Context, identifier any) (
 		}
 	}
 
+	obligations := make([]*policy.Obligation, 0)
+	if av.Obligations != nil {
```

Comment:
if your `unmarshalObligations` helper already handles nil bytes passed in, could this be simplified at all?